command_server/generic/base.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import json
+import aiohttp
+
+
+class BaseView(object):
+
+    @classmethod
+    def as_view(cls):
+        view = cls()
+        return view
+
+    @asyncio.coroutine
```

Comment:
As I know, if you are going use non-blocking IO inside some `def` you should provide `@asyncio.coroutine` for full call-stack of handler